File: backend-api/src/DTOs/OrdersListDTO.cs
namespace unipos_basic_backend.src.DTOs
{
    public sealed class OrdersListDTO
    {
        public Guid Id { get; set; }
        public string CustomerName { get; set; } = string.Empty;
        public string CustomerPhone { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public int TotalQty { get; set; }
        public decimal TotalPay { get; set; }
        public decimal TotalPaid { get; set; }
        public decimal TotalChange { get; set; }
        public OrderStatusEnum Status { get; set; }
        public string Operator { get; set; } = string.Empty;
        public DateTime CreatedAt { get; set; }
    }

    public enum OrderStatusEnum
    {
        Cancelled,
        Pending,
        Paid
    }
}

File: frontend/src/app/_interfaces/orders-list-dto.ts
export interface OrdersListDTO {
  "id": string
  "customerName": string
  "customerPhone": string
  "description": string
  "totalQty": number
  "totalPay": number
  "totalPaid": number
  "totalChange": number
  "status": OrderStatusEnum
  "operator": string
  "createdAt": Date
}

export enum OrderStatusEnum {
  Cancelled,
  Pending,
  Paid
}
